tests/Session/test_session.py

```diff
@@ -0,0 +1,14 @@
+from src.Session.session import Session
+from src.Constants.session_messages import *
+from unittest.mock import patch
+
+import unittest
+
+class Test_Session(unittest.TestCase):
+    def setUp(self):
+        self.session = Session()
+
+    @patch('src.Session.session.input', create=True)
+    def testStart(self, mocked_input):
+        # TODO: This can't be tested right now until we do regression tests
```

Comment:
What do mean by regression tests? o:

Reply:
so basically that would kinda be doing a walkthrough of actually using our application. I don't have the "redirect" part completed yet, so `start()` essentially just starts the program but if you type any commands, it doesn't do anything or call any methods from `commands`. So basically, we kinda have to wait to test this at the end when we can do a full demo of like a person creating a profile, creating a fridge, adding food, etc etc.